app/src/main/res/drawable/defaut.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="32dp" android:viewportHeight="500.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.defaut` appears to be unused
 Unused resources make applications larger and slow down builds.